readme.md

```diff
@@ -54,3 +64,12 @@ Your contributions may be rejected if:
 - The resource is not accessible to everyone (i.e. it requires a subscription).
 - The tool is not maintained or is outdated.
 - The tool is not useful or relevant to the community, or it has very few active users.
+
+## Acknowledgments
+
+This list has been mostly compiled by the moderators of [osu! Game Community Discord server](https://discord.gg/osu) and maintained by the [contributors of this repository](https://github.com/osucord/resources/graphs/contributors).
```

Comment:
Its more the other way around is it not? Maintained by the mods and contributed to by the server *for* the server.

Reply:
Feel free to suggest better wording

Reply:
This works better imo
```suggestion
This list is primarily maintained by the staff of the [osu! Game Community Discord server](https://discord.gg/osu), with contributions from our community. The list of users who have directly contributed to this repository can be found [here](https://github.com/osucord/resources/graphs/contributors).
```
Not sure we really need to link the contributors considering they are on the sidebar on the repository anyway.

Reply:
Hows the edit?

Reply:
LGTM